CONTRIBUTING.md

```diff
@@ -1,8 +1,6 @@
 # Contributing to prebuild
 
-* no commits direct to master
-* all commits as pull requests (one or several per PR)
-* each commit solves one identifiable problem
-* never merge one's own PRs, another contributor does this
-
-
+- no commits direct to master
+- all commits as pull requests (one or several per PR)
+- each commit solves one identifiable problem
+- never merge one's own PRs, another contributor does this
```

Comment:
These lines actually comes from @hintjens and I think we should honor it :)

Reply:
Haha OK